### ISSUE TITLE
Virtual currency of Tree of Savior

### ISSUE THREAD

**Issue by Sunaries** [Treeofsavior/EnglishTranslation#504]
Hey everyone,

In the Korean server the virtual currency is known as NXP. This probably refers to Nexon Point since Nexon is the publisher in Korea. With NXP you can buy items from the Cash Shop. 
I noticed in the latest files that this was changed from NXP to Coin. I did not know that @imcgames made this change.

What do you think of Coin being the term for the virtual currency? Or is there a better term we can come up with for the International server? 

Regards,

**Comment by DrBillygoat** [comment]
I'm partial to "Tin Cans" but I doubt the humans would ever spend such a delicious currency.

But seriously.  Since the currency is used to make items and boosts magically appear out of thin air (thus shouldn't rightfully exist in-game from a lore perspective), it seems odd to purchase them with something as simple as a gold coin.  Unless of course gold doesn't exist in the game world and this mystical element is granted as a symbol of favor from the goddesses for stealing my moms credit card.    Also it sounds a but unintuitive for them to have such a similar name to the game's actual currency, Silver Coins.  

Even before I found out they were gold coins, I thought the currency "Coin" sounded kinda off.  Maybe they should just take a page out of NC Soft's book and call it "IMCoin."

**Comment by vegax87** [comment]
Yeah, IMCoin it's perfect :+1: 
at least IMCoins are not confused with gold coins that will be added in a future patch or expansion..

**Comment by Sunaries** [comment]
IMCoin sounds great and is a lot better than plainly Coin. :+1:

**Comment by ensata** [comment]
I'd prefer it if you didn't break the fourth wall, thanks. These days, a lot of MMOs with this system does at least the token effort of giving it an in-universe name, or even give a basic backstory. Since this game is using classic Silver for money, they can probably just call it Platinum or Electrum coins or something.

Anyway, I can guess pretty quickly that 'Coin' is just a publisher-agnostic temporary name, for demo purposes. You probably shouldn't touch it.

I'm pretty sure that the name of this thing isn't fully up to us, or IMC. If IMC finds a publisher for a region, the name might be whatever that company names their credit points. If IMC self-publishes, (which I'm not fully sure about, since IMC doesn't seem to have a western office yet...) then calling it IMCoin might be back on the table, but market research will probably tell them what I just said: users won't like it when it's stupidly transparent, and you should at least take a token effort to give it a proper name to make it easier to ignore, if they don't want it.

**Comment by DrBillygoat** [comment]
I do see your point about not having a publisher or it not being up to them.  Though most of the other F2P MMOs I've played (actually all I think) have blatantly broken the fourth wall and it never bothered anyone one bit.  Then again when I made my suggestion I forgot that there's an in-game option to gift your "Coin" to other players so I suppose if it's easy to exchange with others it may make more sense this way.
